snooty/rstspec.toml

```diff
@@ -1320,6 +1320,15 @@ type = {link = "https://www.mongodb.com/docs/v5.0%s", ensure_trailing_slash = tr
 [role."v5.1"]
 type = {link = "https://www.mongodb.com/docs/v5.1%s", ensure_trailing_slash = true}
 
+[role."v5.2"]
+type = {link = "https://www.mongodb.com/docs/v5.2%s", ensure_trailing_slash = true}
+
+[role."v5.3"]
+type = {link = "https://www.mongodb.com/docs/v5.3%s", ensure_trailing_slash = true}
+
+[role."rapid"]
+type = {link = "https://www.mongodb.com/docs/rapid%s", ensure_trailing_slash = true}
```

Comment:
```suggestion
# Will link to the latest rapid release and will not be pinned to a specific manual version
[role."rapid"]
type = {link = "https://www.mongodb.com/docs/rapid%s", ensure_trailing_slash = true}
```

Minor suggestion to add a comment re: where this link will target - context seems obvious for everyone familiar, I think, but extra context for those who aren't familiar. 

Let me know if this comment makes sense!